src/utils/api.js

```diff
@@ -0,0 +1,52 @@
+import axios from 'axios';
+import config from './config';
+// import { getToken } from './auth';
+
+
+// const token = getToken();
+const token = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6MSwidXNlcm5hbWUiOiJpbWlyZSIsImVtYWlsIjoiaW1pcmVAZ21haWwuY29tIiwiaWF0IjoxNTQxNTgyMjQzLCJleHAiOjE1NDE1ODk0NDN9.KqR2wO0Q94VjC3zr7CqwYYz-_6HrQY0j63jiBUILlcw';
+
+const uri = config.BASE_URL;
+
+const headers = {
+  'Content-Type': 'application/json',
+  Authorization: `token ${token}`,
+};
+
+const api = {
```

Comment:
Make this axios instance more reusable, use an axions instance for this for example
```
const axiosInstance = axios.create({
  baseURL: {{url}},
  headers: {
    Authorization: `Bearer ${token}`,
  },
});
```

Reply:
updated this